src/App.tsx

```diff
@@ -2,108 +2,48 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { Post } from './types/Post';
+import { FullPost } from './types/FullPost';
+import { User } from './types/User';
+import { PostList } from './components/PostList';
+import { Comment } from './types/Comment';
+
+function getComments(postId: number): Comment[] {
```

Comment:
You can use arrow function for better readability and consistency, like this: `const getComments = (postId: number): Comment[] => { ... }` 🚀